src/web_context.rs

```diff
@@ -33,12 +34,17 @@ pub struct WebContext {
 impl WebContext {
   /// Create a new [`WebContext`].
   ///
+  /// `cache_directory`:
+  /// * Whether the WebView window should have a custom cache path. This is useful in Windows
```

Comment:
`Windows` in the data_directory comment is actually about the Windows OS so here it would be incorrect.